api/index.js

```diff
@@ -97,92 +185,99 @@ app.get('/proof/:trackingId', async (req, res) => {
       }
-      
+
       throw err;
     }
 
     console.log(`got response: ${util.inspect(result)}`);
-    res.json({ result });
-  }
-  catch (err) {
-    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({ error: err.message });
+    res.json({
+      result
+    });
+  } catch (err) {
+    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
+      error: err.message
+    });
   }
 
 });
 
 
-app.get('/key/:keyId', async (req, res) => {
+app.get('/key/:keyId', async(req, res) => {
   try {
-
     req.checkParams('keyId', 'Invalid keyId').notEmpty();
     var errors = await req.getValidationResult();
     if (!errors.isEmpty()) {
-      return res.status(HttpStatus.BAD_REQUEST).json({ error: `there have been validation errors: ${util.inspect(errors.array())}` });
+      return res.status(HttpStatus.BAD_REQUEST).json({
+        error: `there have been validation errors: ${util.inspect(errors.array())}`
+      });
     }
 
-    if (!req.headers['user-token']){
-        return res.status(HttpStatus.BAD_REQUEST).json({ error: `user-token request header is missing` });
+    if (!req.headers['user-token']) {
+      return res.status(HttpStatus.BAD_REQUEST).json({
+        error: `user-token request header is missing`
+      });
     }
-    
+
     // keyId is encoded. leave it encoded since we also use it as part of the URL in the request
     var keyId = req.params.keyId;
-     if (decodeURIComponent(keyId) === keyId) {
+    if (decodeURIComponent(keyId) === keyId) {
       keyId = encodeURIComponent(keyId);
     }
 
-    var userId =await getUserId(req.headers['user-token']);
+    var userId = await getUserId(req.headers['user-token']);
 
     var path = iberaServicesEndpoint + `/api/key/${keyId}?userId=${userId}`;
     var result = await request.get(path, {
-       json: true 
+      json: true
     });
 
     console.log(`got response: ${util.inspect(result)}`);
     res.json(result);
-  }
-  catch (ex) {
-    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({ error: err.message });
+  } catch (ex) {
+    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
+      error: err.message
+    });
   }
 });
 
 
 // TODO: revisit code
 app.get('/hash', (req, res) => {
   console.log(`in hash api- ${util.inspect(req)}`);
-
   var url = decodeURIComponent(req.query.url);
   console.log(`getting url: '${url}'`);
-
   return http.get(url.parse(req.query.url), res => {
     var data = [];
-
-    return res.on('data', function(chunk) {
-      data.push(chunk);
-    })
-    .on('end', function() {
-      //at this point data is an array of Buffers
-      //so Buffer.concat() can make us a new Buffer
-      //of all of them together
-      var buffer = Buffer.concat(data);
-      res.send({result: sha256(buffer).ToUpperCase(), error: result.error});
-    })
-    .on('error', err => {
-      return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({ error: err.message });
-    });
+    return res.on('data', function (chunk) {
```

Comment:
start .on( in a new line as it was before...
this is more readable...

Reply:
done